Lib/python/pyhead.swg

```diff
@@ -75,7 +75,20 @@ SWIG_Python_str_FromChar(const char *c)
 #define Py_hash_t long
 #endif
 
-#ifdef Py_LIMITED_API
+#ifdef Py_GIL_DISABLED
+# undef PyTuple_GET_ITEM
+# undef PyTuple_SET_ITEM
+# undef PyTuple_GET_SIZE
+# undef PyCFunction_GET_FLAGS
+# undef PyCFunction_GET_FUNCTION
+# undef PyCFunction_GET_SELF
+# undef PyList_GET_ITEM
+# undef PyList_SET_ITEM
+# undef PySliceObject
+#endif
+
```

Comment:
Do you have a reference or reason for undefiningthese?

Reply:
https://py-free-threading.github.io/porting/ states that some low level macros like PyList_SET_ITEM aren't safe with free threading. Hence I've undef the macros and redefined them using the limited api version in the next define section.